installer/dockerfile/scheduler/Dockerfile

```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+FROM golang:1.17 AS builder
+WORKDIR /go/src/volcano.sh/
+ADD . volcano
+RUN cd volcano && make vc-scheduler
 FROM alpine:latest
 
-ADD vc-scheduler /vc-scheduler
+COPY --from=builder /go/src/volcano.sh/volcano/_output/bin/vc-scheduler /vc-scheduler
```

Comment:
the bin path is error, `/go/src/volcano.sh/volcano/_output/bin/vc-scheduler` should be:

```
/go/src/volcano.sh/volcano/_output/bin/linux/amd64/vc-scheduler
/go/src/volcano.sh/volcano/_output/bin/linux/arm64/vc-scheduler
```

Reply:
I think here is also the reason of why ci failed.